default.json

```diff
@@ -54,6 +54,20 @@
         "gomodUpdateImportPaths",
         "gomodTidy"
       ]
+    },
+    {
+      "description": "Override the docker:pinDigests preset provided by config:best-practices",
+      "matchDatasources": [
+        "docker"
```

Comment:
```suggestion
        "dockerfile","github-action"
```

Reply:
I kept them separate intentionally, since they are intended to match and revert separate presets.  I can certainly combine them if you really prefer that.  Let me know.